### ISSUE TITLE
A/D and D/A block revisions

### ISSUE THREAD

**Issue by kgracey** [parallaxinc/BlocklyProp#500]
@zfi this is everything you need to know to clean up the A/D blocks in the Voltage menu. 

1. There are currently two blocks doing the same thing - delete the second one:
![screen shot 2016-08-11 at 11 37 17 am](https://cloud.githubusercontent.com/assets/9534148/17599837/d6382b5a-5fb5-11e6-9471-3280628479e6.png)
![screen shot 2016-08-11 at 11 37 59 am](https://cloud.githubusercontent.com/assets/9534148/17599838/d63cacc0-5fb5-11e6-9800-c729e9b1421b.png)
2. Change the text on the block to be "0-5V ADC reading in Volts x 100". No changes to the 0-3 CH drop-down menu. 
3. The code generated by the A/D block is below):
```
#include "simpletools.h"
#include "abvolts.h"

int item;

int main()
{
    while(1)
    {
        item = ad_in(1);
        item = item * 500 / 4096;
    }

}
```
In summary, I just added a second line that scales the code to 0-5V output in whole numbers.

**Comment by zfi** [comment]
Has this been submitted to the demo branch?

**Comment by DarkmatterVale** [comment]
@zfi I have it completed but I cannot test locally because I am away from my main dev setup. Should I send to demo without testing locally?

**Comment by zfi** [comment]
Yes, send it to demo with Needs Verification label.

**Comment by zfi** [comment]
Final block looks like this:

![2016-08-18_14-56-32](https://cloud.githubusercontent.com/assets/8442260/17792141/0db2fea4-6554-11e6-854e-dd5ed6c0384b.png)
